CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Fixed updates to mutable state during subscribe leading to non-existant/frozen video streams.
+- Fixed inconsistent default maxBitrate values in the NScaleVideoUplinkBandwithPolicy constructor leading to the default ideal max bitrate not being honored.
```

Comment:
We are prepping for the JS SDK release, hence can you please add a new section as this change is newer than the release going changes.

```
## [Unreleased]

### Added

### Removed

### Removed

### Fixed
- Fixed inconsistent default maxBitrate values in the NScaleVideoUplinkBandwithPolicy constructor leading to the default ideal max bitrate not being honored.

### Changed
```